locale/eu/ES/pseudomap.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Should we treat eu-ES and ps-AF locales are pseudo locales?
To match loctool, I think these locales shouldn't be pseudo locales. no?

Reply:
updated, please re-review